packages/design-tokens/src/scss/generated/_colors.scss

```diff
@@ -1,4 +1,5 @@
 // Generated Colors from Supernova. Do not edit manually.
+$01-default: #17171700 !default;
```

Comment:
Here applies the same as in Typography. Please define colours only in Figma, as you probably already need them there and if you define them only in Supernova, they are missing some structural info we need in exporter to name and place them correctly.